### ISSUE TITLE
Insecure protocol with repository - maven7(http://maven.getsocial.im/)

### ISSUE THREAD

**Issue by NorrisWu0** [getsocial-im/getsocial-react-native-sdk#35]
### Issue Info

There was no issue building the project with gradle 4.2.2, but one of the dependencies we used requires us to upgrade gradle version to 7.0.3. Building the project with upgraded build tool will trigger below error.

### Error Info
```
Error: Command failed: gradlew.bat app:installDebug -PreactNativeDevServerPort=8081

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':getsocial-react-native-sdk:compileDebugLibraryResources'.
> A failure occurred while executing com.android.build.gradle.tasks.CompileLibraryResourcesTask$CompileLibraryResourcesAction       
   > Could not isolate value com.android.build.gradle.tasks.CompileLibraryResourcesTask$CompileLibraryResourcesParams_Decorated@29ab4072 of type CompileLibraryResourcesTask.CompileLibraryResourcesParams
      > Could not resolve all dependencies for configuration ':getsocial-react-native-sdk:detachedConfiguration1'.
         > Using insecure protocols with repositories, without explicit opt-in, is unsupported. Switch Maven repository 'maven7(http://maven.getsocial.im/)' to redirect to a secure protocol (like HTTPS) or allow insecure protocols. See https://docs.gradle.org/7.0.2/dsl/org.gradle.api.artifacts.repositories.UrlArtifactRepository.html#org.gradle.api.artifacts.repositories.UrlArtifactRepository:allowInsecureProtocol for more details.
```

### Steps to reproduce
1. Update `android/build.gradle` with
  ```
  buildscript {
    ...
      dependencies {
      ...
      classpath 'com.android.tools.build:gradle:7.0.3'
    }
  }  
```
2. Run `yarn android`
3. Build failed with error.

**Comment by demiandenker** [comment]
Hi @NorrisWu0, you should be able to quickly fix this issue by looking for the `build.gradle` file inside the getsocial sdk > android folder and changing `http` to `https` on this line:
https://github.com/getsocial-im/getsocial-react-native-sdk/blob/ff0eaca74f9c823d322f563c6624d91d784a1df3/sdk7demo/getsocial-react-native-sdk/android/build.gradle#L27

**Comment by NorrisWu0** [comment]
Hi @demiandenker , sorry for not getting back to you earlier and yes the suggested code change did fix the issue for us. Any chance we will see this being updated in the package?

**Comment by demiandenker** [comment]
@NorrisWu0  It has already been addressed in the latest RN SDK v7.9.1 released about 2 weeks ago.
